### ISSUE TITLE
Connector should verify notification from Ledger

### ISSUE THREAD

**Issue by naoitoi** [interledger/five-bells-connector#105]
This issue is a follow up to https://github.com/interledger/five-bells-connector/issues/94

To fix RILP-254, 
(1) Connector should not trust any transfer (e.g. the ones from Sender).  It should only trust the notification from Ledger. 
(2) Connector should verify the notification from Ledger, e.g., via digital signature. 

Issue 94 fixes (1).  
This new issue is for (2).

**Comment by clark800** [comment]
Suggest rewording title to "Connector should verify notification from Ledger"

**Comment by naoitoi** [comment]
Discussed with @justmoon  and we agreed that ilp-server team should implement this.  
I will close this issue and file a ticket for ilp-server team.
